bin/printenv.ml

```diff
@@ -75,7 +75,11 @@ let term =
             User_error.raise
               [ Pp.text "Environment is not defined in install dirs" ])
       in
-      Build_system.run_exn (fun () -> Build_system.build request) >>| function
+      Build_system.run_exn (fun () ->
+          let open Memo.Build.O in
+          let+ res, _facts = Action_builder.run request Eager in
```

Comment:
Should we make this `Lazy`? (or leave a CR-someday)
It seems pretty unlikely that `Eager` is deliberate.